collector/compile-benchmarks/typenum-1.17.0/src/lib.rs

```diff
@@ -69,11 +69,11 @@
 use core::cmp::Ordering;
 
 mod generated {
-    include!(concat!(env!("OUT_DIR"), "/op.rs"));
-    include!(concat!(env!("OUT_DIR"), "/consts.rs"));
+    include!("generated/op.rs");
```

Comment:
I think it would be useful to add comments here explaining the changes made for the importing.

Reply:
Done.